website/docs/components/embeddings/model2vec.md

```diff
@@ -0,0 +1,154 @@
+---
+title: 'Model2Vec Embedding Models'
+sidebar_label: 'Model2Vec'
+sidebar_position: 4
+---
+
+To use a Model2Vec embedding model with Spice, specify the `model2vec` prefix in the `from` field of your configuration. Model2Vec is a technique that distills embeddings from transformer models into static word embeddings, providing efficient embedding generation, in parallel, without performing external API calls.
```

Comment:
Start with what model2vec is and then how to use it. A link to the HuggingFace blog is probably good too. https://huggingface.co/blog/Pringled/model2vec